### PR TITLE
Add mkldnn bfloat option in inference scripts

### PR DESCRIPTION
**Pull request by lidanqing-vv**
Add mkldnn bfloat option in inference scripts

### CODE REVIEW

File: configs/runtime.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use_gpu: true
+use_gpu: false
```
Comment (nemonameless) [inline configs/runtime.yml]: revert

Comment (lidanqing-vv) [reply configs/runtime.yml]: Hi, thanks for reviewing, this line is changed now. Please continue your review. Thank you very much!

**Review by nemonameless — DISMISSED** [review]

**Comment by lidanqing-vv** [comment]
@jakpiase Hi, please review this PR. Thanks.

**Review by jakpiase — DISMISSED** [review]
LGTM

**Comment by lidanqing-vv** [comment]
@wozna Hi, Joanna, could you please review this PR, Thanks !

File: static/deploy/python/infer.py [hunk 2]
```diff
@@ -557,15 +566,16 @@ def main():
...
+        enable_mkldnn=FLAGS.enable_mkldnn,
+        enable_mkldnn_bfloat16=FLAGS.enable_mkldnn_bfloat16)
     if config.arch == 'SOLOv2':
         detector = DetectorSOLOv2(
             config,
             FLAGS.model_dir,
             device=FLAGS.device,
             run_mode=FLAGS.run_mode,
             trt_calib_mode=FLAGS.trt_calib_mode,
-            enable_mkldnn=FLAGS.enable_mkldnn)
```
Comment (wozna) [inline static/deploy/python/infer.py:568]: Here you removed enable_mkldnn, and I think both enable_mkldnn and enable_mkldnn_bfloat16 should remain

File: static/deploy/python/infer.py [hunk 3]
```diff
@@ -229,15 +232,17 @@ def __init__(self,
...
         super(DetectorSOLOv2, self).__init__(
             config=config,
             model_dir=model_dir,
             device=device,
             run_mode=run_mode,
             threshold=threshold,
             trt_calib_mode=trt_calib_mode,
-            enable_mkldn=enable_mkldnn)
+            enable_mkldn=enable_mkldnn,
+            enable_mkldnn_bfloat16=FLAGS.enable_mkldnn_bfloat16)
```
Comment (wozna) [inline static/deploy/python/infer.py]: Is the parameter directly from FLAGS `FLAGS.enable_mkldnn_bfloat16` used here on purpose? Shouldn't we use just the `enable_mkldnn_bfloat16` argument from the parent class defined above?

File: static/deploy/python/infer.py (hunk 2 above)
Comment (lidanqing-vv) [reply static/deploy/python/infer.py:568]: Hi, you are right, I have changed it.

**Review by wozna — DISMISSED** [review]
LGTM

**Review by jakpiase — DISMISSED** [review]
LGTM

**Comment by lidanqing-vv** [comment]
@nemonameless Could you please continue your review? Two colleagues have reviewed and I also have tested this PR using frcnn-r50-fpn retinanet-fpn bf16 inference on Intel machines. Thanks.

**Review by nemonameless — DISMISSED** [review]

**Comment by nemonameless** [comment]
由于deploy/python/infer.py 代码升级，请更新一下PR谢谢~

**Comment by lidanqing-vv** [comment]
Hi, @nemonameless I have resolved the conflict and also ran locally successfully. Could you please review again? Thanks !

**Review by nemonameless — APPROVED** [review]

**Comment by lidanqing-vv** [comment]
@baoachun Hi, could you please ask Baidu colleague to add one more approval to merge this PR? This is needed for SPR bfloat16 for your validation. (ppyolo_r50, picodet and PLCNet bfloat16 by this PR)

**Comment by lidanqing-vv** [comment]
@baoachun Please help merge this PR, thank you

**Review by jerrywgz — APPROVED** [review]

**Pull request merged**